### PR TITLE
Implement Login Widget & Translation Ready for WordPress.org

### PR DESCRIPTION
**Pull request by mfaizanrasool**
Implement Login Widget Area.

### CODE REVIEW

File: admin/class-wpfep-system-status.php [hunk 1]
```diff
@@ -50,57 +50,57 @@ public static function status_report()
...
 						<td><?php echo esc_html($environment['home_url']); ?></td>
 					</tr>
 					<tr>
-						<td data-export-label="<?php esc_attr_e('Site URL', 'wpfep'); ?>"><?php esc_attr_e('Site URL', 'wpfep'); ?>:</td>
+						<td data-export-label="<?php esc_attr_e('Site URL', 'wp-front-end-profile'); ?>"><?php esc_attr_e('Site URL', 'wp-front-end-profile'); ?>:</td>
 						<td><?php echo esc_html($environment['site_url']); ?></td>
 					</tr>
 					<tr>
-						<td data-export-label="<?php esc_attr_e('wpfep version', 'wpfep'); ?>"><?php esc_attr_e('wpfep version', 'wpfep'); ?>:</td>
+						<td data-export-label="<?php esc_attr_e('wp-front-end-profile version', 'wp-front-end-profile'); ?>"><?php esc_attr_e('wp-front-end-profile version', 'wp-front-end-profile'); ?>:</td>
```
Comment (musamamasood) [inline admin/class-wpfep-system-status.php:79]: @mfaizanrasool `<?php esc_attr_e('wp-front-end-profile version', 'wp-front-end-profile'); ?>">` should be `<?php esc_attr_e('WP Frontend Profile version', 'wp-front-end-profile'); ?>">`

File: inc/class-wpfep-login-widget.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        $defaults = array(
+            'title' => __('Login', 'wp-front-end-profile'),
+        );
+
+        // Parse current settings with defaults.
+        extract(wp_parse_args((array) $instance, $defaults)); ?>
+
+		<?php // Widget Title.?>
+			<p>
+			<label for="<?php echo $this->get_field_id('title'); ?>"><?php _e('Title:', 'wp-front-end-profilep'); ?></label>
```
Comment (musamamasood) [inline inc/class-wpfep-login-widget.php:31]: @mfaizanrasool incorrect text-domain `wp-front-end-profilep`

File: inc/class-wpfep-login-widget.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<?php
+
+// Main constructor
+// The widget class.
```
Comment (musamamasood) [inline inc/class-wpfep-login-widget.php:4]: Add PHPDoc comments

File: admin/class-wpfep-system-status.php (hunk 1 above)
Comment (mfaizanrasool) [reply admin/class-wpfep-system-status.php]: Mentioned changes are Fixed.

File: inc/class-wpfep-login-widget.php (hunk 2 above)
Comment (mfaizanrasool) [reply inc/class-wpfep-login-widget.php]: Mentioned changes are Fixed

File: inc/class-wpfep-login-widget.php (hunk 3 above)
Comment (mfaizanrasool) [reply inc/class-wpfep-login-widget.php:6]: Mentioned changes are Done.

**Pull request merged**